core/src/main/java/org/elasticsearch/search/aggregations/bucket/MultiBucketsAggregation.java

```diff
@@ -64,6 +66,12 @@
 
         Object getProperty(String containingAggName, List<String> path);
 
+        @Override
+        default void readFrom(StreamInput in) throws IOException {
+            // NORELEASE remove me when no Buckets override it
+            throw new UnsupportedOperationException("Prefer the Writeable interface");
+        }
```

Comment:
I'm fairly sure we don't need it here because I only implemented `Writeable` instead of `NamedWriteable`. I'll remove it.